tests/test_plotting.py

```diff
@@ -0,0 +1,90 @@
+import oogeso
+from oogeso import plots as op
+from oogeso import dto
+import pandas as pd
+
+
+def test_plot_co2_intensity(leogo_expected_result: dto.SimulationResult):
```

Comment:
We need to add a check to see if plotting is installed. Add this decorator and include checks for all relevant plotting modules.:
import sys
@pytest.mark.skipif(not "matplotlib" in sys.modules, reason="Plotting modules has not been installed)
def test_plot_x_y_z...

Reply:
`@pytest.mark.skipif(not all([["matplotlib", "plotly", "seaborn"] in sys.modules]), reason="Plotting modules has not been installed")`